### DISCUSSION TITLE
Hardware specifications used

### DISCUSSION THREAD

**Discussion by jennyzzt** [labmlai/annotated_deep_learning_paper_implementations#151] [General] [answered]
Hi, I am interested in fine-tuning GPT-NeoX. What was the hardware specifications that you used and how long did it take you?
Would 2 servers of 8 A100s GPU be sufficient?

**Comment by Siimarras** [comment, marked as answer]
I'm using Pocophone F1 mostly.
I3 2350M CPU if not mistaken.
A bit older model, but still fully functional.
